src/arrayMethodSort.js

```diff
@@ -5,7 +5,24 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    const start = this.slice();
```

Comment:
The 'start' variable is not very descriptive. Consider renaming it to something more meaningful, like 'copiedArray' or 'sortedArray'. And BTW this variable seems redundant, you can use this() instead. The sort method is mutable